### ISSUE TITLE
[SpeedySpeech] VAL AVG LOSS: NaN

### ISSUE THREAD

**Issue by ductho9799** [mozilla/TTS#695]
I tried training the Speedy Speech for the Vietnamese dataset not using phoneme. After 5000 steps, the loss is increased dramatically.
![Screenshot from 2021-04-20 10-42-06](https://user-images.githubusercontent.com/72780076/115334187-1f424c80-a1c5-11eb-861b-24421a11fc97.png)
Have anyone trained the SpeedySpeech and faced this problem yet?

**Comment by erogol** [comment]
in general such problems are dataset specific and hard to replicate.

two suggestions

- disable mixed_precision training
- try a smaller learning rate.

if neither of them works, then you should debug and see what (which layer or data sample etc.) causes the problem

**Comment by erogol** [comment]
btw for the latest version of TTS I suggest you moving to [coquiTTS](https://github.com/coqui-ai/TTS)

**Comment by ductho9799** [comment]
Thank you for your suggestions and your open source.  
Do you have a plan for the experiments about multilingual speech synthesis in the future?

**Comment by erogol** [comment]
We have an active project on it. You can join in if you like. We have an open Gitter channel for that project too.
